vm/src/slots.rs

```diff
@@ -40,7 +42,7 @@ pub struct PyClassSlots {
     pub flags: PyTpFlags,
     pub name: PyRwLock<Option<String>>, // tp_name, not class name
     pub new: Option<PyNativeFunc>,
-    pub call: Option<PyNativeFunc>,
+    pub call: AtomicCell<u128>,
```

Comment:
does anyone have idea using neither lock nor unsafe?

Reply:
Just `fn(&VirtualMachine, PyFuncArgs) -> PyObjectRef`? Could generate it in pyslot like
```rust
slots.#slot_ident = Some(
    |vm: &::rustpython_vm::VirtualMachine, args: ::rustpython_vm::function::PyFuncArgs| -> ::...::PyResult {
        <::rustpython_vm::function::IntoPyNativeFunc<_>>::call(&Self::#ident, vm, args)
    } as _
);
```

Reply:
thank you! it seems working